odl/discr/discr_mappings.py

```diff
@@ -63,20 +63,20 @@ def __init__(self, map_type, fset, grid, dspace, order='C', linear=False):
         fset : `FunctionSet`
             The undiscretized (abstract) set of functions to be
             discretized
-        grid :  `TensorGrid`
-            The grid on which to evaluate. Must be contained in
-            the common domain of the function set.
+        partition : `RectPartition`
+            Partition of (a subset of) ``fset.domain`` based on a
```

Comment:
I'll check. I don't remember putting in a test explicitly for this, but since the weighting is based on the partition only, it should always give the correct results.